modules/backendscheduler/backendscheduler.go

```diff
@@ -255,6 +259,11 @@ func (s *BackendScheduler) Next(ctx context.Context, req *tempopb.NextJobRequest
 
 		return resp, nil
 	case <-timeoutCtx.Done():
+		// Add a jitter after timeout so all workers don't all retry at the same time
```

Comment:
In thinking about the conditions for this to help, I think in both the scheduelr crash condition and the normal operations, jitter would be better applied to the backendworker.

The only think the scheduler can do if it runs out of memory or panics is crash, which would signal a disconnect at the worker and immediately retry (after the 100ms initial delay).  I believe the backoff is deterministic in its delay incrementing, so jitter here would help the crash case, but also potentially the normal operations.  

What do you think about moving this to the worker?

Reply:
yeah, we can move this to worker.

Reply:
@zalegrala updated to move the jitter into the worker so now all calls to backendscheduler are with jitter.